src/main/java/com/internet/shop/controllers/orders/ShowUserOrdersController.java

```diff
@@ -24,9 +25,11 @@
     @Override
     protected void doGet(HttpServletRequest req, HttpServletResponse resp)
             throws ServletException, IOException {
-        List<Order> orderList = orderService.getUserOrders(USER_ID);
+        HttpSession session = req.getSession();
+        Long userId = Long.parseLong(String.valueOf(session.getAttribute(USER_ID)));
```

Comment:
Long userId = (Long) req.getSession().getAttribute(USER_ID);
You can change it like that

Reply:
and everywhere else in the order

Reply:
I agree with Nikita, let's use casting here